solr/solrj/build.gradle

```diff
@@ -75,6 +76,7 @@ dependencies {
   testImplementation 'commons-io:commons-io'
 
   testImplementation 'javax.servlet:javax.servlet-api'
```

Comment:
Not sure we need both `'javax.servlet:javax.servlet-api'` and `'org.eclipse.jetty.toolchain:jetty-servlet-api'`? I need to check this.

Reply:
I see there is a comment thread elsewhere on this in the PR.

Reply:
This is addressed now.